README.md

```diff
@@ -131,6 +131,47 @@ However, Lovelace provides a sensible `#comment` function you can use:
 ![comment](examples/comment.png)
 
 
+### Raw Text
+You can use raw text syntax in typst to simplify the syntax of pseudocode.
+
+We use `'''` to escape the markdown code block syntax here, and you can look directly at [the source file](./examples/rawtext.typ).
+
+```typ
+#show: setup-lovelace
+
+#let redbold(it) = {
+  set text(fill: red, weight: "bold")
+  it
+}
+
+#let pseudocode = pseudocode.with(scope: (redbold: redbold))
+
+#algorithm(
```

Comment:
Maybe actually don't use `#algorithm` here, it is not necessary for a minimal example.